tests/e2e-playwright/steps/api.ts

```diff
@@ -0,0 +1,67 @@
+import { config } from '../../e2e/config.js'
+import { UsersEnvironment } from '../../e2e/support/environment'
+import { api } from '../../e2e/support'
+import { ResourceType } from '../../e2e/support/api/share/share'
+
+export async function userHasBeenCreated(
+  usersEnvironment: UsersEnvironment,
+  stepUser: string,
+  userToBeCreated: string
+): Promise<void> {
+  const admin = usersEnvironment.getUser({ key: stepUser })
+  const user = usersEnvironment.getUser({ key: userToBeCreated })
+  // do not try to create users when using predefined users
+  if (!config.predefinedUsers) {
+    await api.provision.createUser({ user, admin })
+  }
+}
+
+export async function createFolderInPersonalSpaceUsingAPI(
```

Comment:
it's better to follow same naming for all the step methods 
```suggestion
export async function userHasCreatedFolder(
```